operator/src/lib.rs

```diff
@@ -875,11 +919,22 @@ pub async fn create_controller(client: Client) {
         .owns(cmd_start_api, ListParams::default())
         .owns(cmd_stop_api, ListParams::default());
 
+    let product_config_path = cli::product_config_path(
+        "spark-operator",
+        vec![
+            "deploy/config-spec/properties.yaml",
+            "/etc/stackable/spark-operator/config-spec/properties.yaml",
+        ],
+    )?;
+
     let product_config =
-        ProductConfigManager::from_yaml_file("deploy/config-spec/properties.yaml").unwrap();
+        ProductConfigManager::from_yaml_file(product_config_path.as_str()).unwrap();
```

Comment:
I believe the CLI handling should not happen in the library itself. This needs to move to the server crate.